solidity/test/LiquidityLayerRouter.t.sol

```diff
@@ -95,7 +95,7 @@ contract LiquidityLayerRouterTest is Test {
             TypeCasts.addressToBytes32(address(destinationBridgeAdapter))
         );
         destinationBridgeAdapter.enrollRemoteRouter(
-            destinationDomain,
+            originDomain,
```

Comment:
How did a test not catch this?

Reply:
we weren't actually using the remote router stuff everywhere in the LL